src/models/Project/createProject/index.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import type { Project } from '@models/Project'
+import type { ValidationError } from '@types/ValidationError'
+import database from '@server/database'
+import ProjectUserModel from '@models/ProjectUser'
+
+export type CreateProjectArgs = {
```

Comment:
Do we all agree then that we use naming convention `ActionArgs` and `ActionResponse` for those types ?

Reply:
Or we switch to `ActionInput` and `ActionPayload`?